### PR TITLE
Update Known working Bluetooth adapters with performance data

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Update Known working Bluetooth adapters

I dropped the `Long Range` language as that didn't really stand up in testing and replaced it with External Antenna which is a clear differentiator but did not always improve range vs some of the better performing adapters.  Range is not considered in the data since its hard to come up with a way to accurately estimate it due to interference/repeatable testing.

The following requirements must be met for an adapter to be labeled as High Performance:

- Establish a connection in about 1s or less
- Process at least one advertisement per second from a device without dropping data
- 95% of connection attempts are successful within two tries
- Meets the above requirements with Home Assistant Core 2022.11.1 or later and Home Assistant Operating System 9.3 or later

Performance testing used the following hardware:

- Active connection to Nanoleaf A19 Bulb NL45-0800 after GATT services were cached by BlueZ
- Advertisements from an Oral-B iO Series 8

I purchased and tested all the current adapters on the list to see which ones performed the best. (More to come but not all have arrived yet)

IMG_0035

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bluetooth.markdown [hunk 1]
```diff
@@ -79,18 +79,41 @@ If you experience an unreliable Bluetooth connection, installing a short USB ext
...
+| SUMEE BT501                       | RTL8761BU  |                  |                   |
+| UGREEN CM390                      | RTL8761BU  |                  |                   |
+| XDO BT802                         | RTL8761BU  |                  | ✅                |
+| ZEXMTE BT-505                     | RTL8761BU  |                  | ✅                |
+| ZEXMTE BT-DG54                    | RTL8761BU  |                  |                   |
+| ZETSAGE BH451A                    | RTL8761BU  |                  |                   |
+
+#### Performance
+
+Performance is primarily determined combination of the chip and the Linux drivers that the adapter uses. Some vendors using the same chip had an unacceptable performance and are listed as unsupported.
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
Performance is primarily determined by a combination of the chip and the Linux drivers for the adapter. Some vendors using the same chip had an unacceptable performance and are listed as unsupported.
```

File: source/_integrations/bluetooth.markdown [hunk 2]
```diff
@@ -79,18 +79,41 @@ If you experience an unreliable Bluetooth connection, installing a short USB ext
...
+
+- Establish a connection in about 1s or less
+- Process at least one advertisement per second from a device without dropping data
+- 95% of connection attempts are successful within two tries
+- Meets the above requirements with Home Assistant Operating System 9.3 or later
+
+Performance testing used the following hardware:
+
+- Active connection to Nanoleaf A19 Bulb NL45-0800 after GATT services were cached by BlueZ
+- Advertisements from an Oral-B iO Series 8
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown:102]:
```suggestion
- Advertisements from an Oral-B iO Series 8
- Home Assistant Blue running Home Assistant Operating System 9.3 (excludes Raspberry Pi on-board)
```

File: source/_integrations/bluetooth.markdown [hunk 3]
```diff
@@ -79,18 +79,42 @@ If you experience an unreliable Bluetooth connection, installing a short USB ext
...
+- Home Assistant Blue running Home Assistant Operating System 9.3 (excludes Raspberry Pi on-board)
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- External Adapters only: Home Assistant Blue running Home Assistant Operating System 9.3
```

**Review by frenck — APPROVED** [review]
Thanks, @bdraco 👍 

That is some really helpful information!

../Frenck

**Pull request merged**